docs/user/trusted-publishers/security-model.md

```diff
@@ -133,7 +133,21 @@ own security model and considerations.
 
 === "Google Cloud"
 
-    TODO
+    ### Security Model
+
+    If a trusted publisher is configured for a given PyPI project, any service
+    that uses the configured service account can request an OpenID Connect token
+    from Google's identity provider on behalf of that identity. That token can be
+    exchanged for a PyPI API token with the ability to publish to the PyPI project.
```

Comment:
Should we mention here the (optional) possiblity of further restricting the service by configuring the subject ID? Just re-stating what is already mentioned in the `add-a-publisher.md` doc:
```suggestion
    exchanged for a PyPI API token with the ability to publish to the PyPI project.
    The identity used for publishing can be optionally constrained further by
    by specifying the subject, an ID that represents the principal making the request.
```